pydantic_ai_slim/pydantic_ai/messages.py

```diff
@@ -947,6 +947,9 @@ class ModelRequest:
     kind: Literal['request'] = 'request'
     """Message type identifier, this is available on all parts as a discriminator."""
 
+    run_id: str | None = None
+    """A unique identifier to identify the run."""
```

Comment:
Let's expand this slightly; same for the one on `ModelResponse`:

```suggestion
    """The unique identifier for the agent run that this request was created in."""
```